changelogs/unreleased/5962-shadialtarsha-small.md

```diff
@@ -0,0 +1,2 @@
+## Configure TypedPerFilterConfig with direct responses
```

Comment:
nit: since this is a small change, can make this a single line with no title

something like `Fixes bug where external authorization configuration was ignored on HTTPRoute rules configured with direct responses.`